data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -351,7 +351,10 @@ then
 			--cloud-credential-operator-image=${CLOUD_CREDENTIAL_OPERATOR_IMAGE}
 
 	cp cco-bootstrap/manifests/* manifests/
-	cp cco-bootstrap/bootstrap-manifests/* bootstrap-manifests/
+	# skip copy if empty directory
+	if find cco-bootstrap/bootstrap-manifests -mindepth 1 -print -quit | grep -q .; then
+		cp cco-bootstrap/bootstrap-manifests/* bootstrap-manifests/
+	fi
```

Comment:
can you switch this to the static pod file maybe instead of this list?

Reply:
can you clarify what you have in mind? are you suggesting putting down a static pod yaml file that just runs "sleep 3600" instead of starting up the CCO binary when CCO is determined to be disabled?